core-bundle/src/Security/ContaoCorePermissions.php

```diff
@@ -106,7 +106,8 @@ final class ContaoCorePermissions
 
     /**
      * Access is granted if the current user can access the form field type.
-     * Subject must be a content element type (e.g. "hidden").
+     * Subject can be a content element type (e.g. "hidden")
+     * or NULL to check if any field type is allowed.
```

Comment:
that would probably work too, but why? Also, it would not work if there actually is a property `*`. Even though I agree thats unlikely, it still could be a conflict.

Reply:
Because using something for "everything" is more intuitive than using nothing for "all". 😄 Also, `*` seems to be a pretty common placeholder for "everything". Think of glob patterns for example.

Reply:
A field cannot start with `*`. And I agree, '*' is more speaking - I'd prefer that too :)

Reply:
As briefly discussed, omitting the subject is better than using `*`.